wrappers/wrapper.rb

```diff
@@ -713,6 +796,10 @@ def check_distances(vrp, unfeasible)
       unfeasible = check_unreachable(vrp, :time, unfeasible)
       unfeasible = check_unreachable(vrp, :distance, unfeasible)
       unfeasible = check_unreachable(vrp, :value, unfeasible)
+      vrp.relations.each{ |relation|
+        # If relation becomes empty it is removed by add_unassigned
+        sequence_relation_with_no_reachable_vehicle?(unfeasible, vrp, relation)
+      }
```

Comment:
It is better if this block comes after the next (vrp.services `no_compatible_vehicle_with_compatible_tw`) block, no? Otherwise, a service might get marked as unfeasible after the relation check and the services "following" this service won't get marked infeasible.